examples/schema_site_contract_service_graph/main.tf

```diff
@@ -0,0 +1,97 @@
+terraform {
+  required_providers {
+    mso = {
+      source = "CiscoDevNet/mso"
+    }
+  }
+}
+
+provider "mso" {
+  username = "" # <MSO username>
+  password = "" # <MSO pwd>
+  url      = "" # <MSO URL>
+  insecure = true
+}
+
+
+data "mso_schema" "schema1" {
+  name = "schema1"
+}
+
+data "mso_schema" "schema2" {
+  name = "schema2"
+}
+
+data "mso_site" "site1" {
+  name = "site1"
+}
+
+data "mso_schema_template" "t1" {
+  name      = "t1"
+  schema_id = data.mso_schema.schema1.id
+}
+
+data "mso_schema_template" "t2" {
+  name      = "t2"
+  schema_id = data.mso_schema.schema2.id
+}
+
+data "mso_schema_template_bd" "bd1" {
+  schema_id     = data.mso_schema.schema1.id
+  template_name = "t1"
+  name          = "bd1"
+}
+
+data "mso_schema_template_bd" "bd2" {
+  schema_id     = data.mso_schema.schema2.id
+  template_name = "t2"
+  name          = "bd2"
+}
+
+data "mso_schema_template_contract" "c1" {
```

Comment:
I think we need a full example with the template contract service graph part in as resources
I would also do the service graph part in it too (at both template and site levels) as resources